packages/dashboard/src/utils/useStoryView.js

```diff
@@ -50,6 +50,7 @@ export default function useStoryView({
   const [authorFilterId, _setAuthorFilterId] = useState(null);
   const [queriedAuthors, setQueriedAuthors] = useState([]);
   const showStoriesWhileLoading = useRef(false);
+  const initialPageReady = useRef(false);
```

Comment:
I don't think I'm completely following. What's the behavior we get from this being a ref?

Reply:
oh just me following the same pattern as loading, updating it to just be state instead.